frontend/__tests__/components/LikeList.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { render, screen } from '@testing-library/react'
+import LikeList from '@/components/LikeList'
+
+const FAKE_RESTAURANT_ONE = {
+  name: 'Fake Restaurant 1',
+  address: '123 Fake St.',
+  cuisine: 'Fakish',
+  rating: 4.5,
+  phone: '2130001111',
+  image_url: 'http://www.dummyrestaurant.com/pic',
+  website_url: 'http://www.dummyrestaurant.com',
+}
+
+const FAKE_RESTAURANT_TWO = {
+  name: 'Fake Restaurant 2',
+  address: '123 Fake St.',
+  cuisine: 'Asian',
+  rating: 4.0,
+  phone: '2130001111',
+  image_url: 'http://www.dummyrestaurant.com/pic',
+  website_url: 'http://www.dummyrestaurant.com',
+}
+
+const FAKE_RESTAURANT_THREE = {
+  name: 'Fake Restaurant 3',
+  address: '123 Fake St.',
+  cuisine: 'American',
+  rating: 3.5,
+  phone: '2130001111',
+  image_url: 'http://www.dummyrestaurant.com/pic',
+  website_url: 'http://www.dummyrestaurant.com',
+}
+
+const LIKED_RESTAURANTS = [FAKE_RESTAURANT_ONE, FAKE_RESTAURANT_TWO, FAKE_RESTAURANT_THREE]
```

Comment:
@seoulection  made a good point on this PR https://github.com/8l-project-jam/tastemate/pull/55#discussion_r1358569798 that the all caps variables are typically for variables that are globally scoped. Can we change these to lower case?